grc/core/params/dtypes.py

```diff
@@ -32,19 +32,18 @@ def decorator(func):
         return func
     return decorator
 
-
 class ValidateError(Exception):
     """Raised by validate functions"""
 
 
 @validates('id')
-def validate_block_id(param):
+def validate_block_id(param,bl):
```

Comment:
little nit: `bt` is not a very (self-)descriptive name, why not use sth. like `black_listed_ids`.